docs/code_ref/flct.rst

```diff
@@ -0,0 +1,7 @@
+****
+FLCT
+****
+
+This subpackage contains routines which can be used to perform Fourier Local Correlation Tracking.
+
```

Comment:
Add in these sentences.  "This subpackage wraps a C-based implementation of the FLCT algorithm.  The C code is provided by G. H. Fisher and B. Welsch.  For further information see Welsch et al, 2004, ApJ, 610 c(doi:10.1086/421767) and Fisher & Welsch 2008, 383, 373, Astronomical Society of the Pacific Conference Series (https://arxiv.org/abs/0712.4289)."